style.css

```diff
@@ -0,0 +1,205 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: absolute;
+  width: 375px;
+  height: 44px;
+  left: calc(50% - 375px / 2);
+  top: 0;
+}
+
+.logo-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 1.25rem;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+}
```

Comment:
- [ ] In my opinion, to apply the DRY rule in CSS, you can take the `<a>` tag directly in your header tag. and give the header tag `display: flex` to align the logo and hamburger menu. by doing this, you can remove a lot of repetitive CSS used in the different Selectors below. I can see a lot of repetitive use of the following lines and similar ones which has no use. Kindly consider making your CSS code DRY.
```
flex: none;
  order: 0;
  flex-grow: 0;
```
Kindly have a look at this link: https://medium.com/circuits-technical-talks/dry-css-9cbe3215af3d